src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -2969,6 +2969,26 @@ default Emote getEmoteById(long id)
     @CheckReturnValue
     RestAction<List<Webhook>> retrieveWebhooks();
 
+    /**
+     * Retrieves the {@link GuildWelcomeScreen Welcome screen} for this Guild.
+     * <br>The welcome screen is shown to all members after joining the Guild.
+     *
+     * <p>Possible {@link net.dv8tion.jda.api.requests.ErrorResponse ErrorResponses} include:
+     * <ul>
+     *     <li>{@link net.dv8tion.jda.api.requests.ErrorResponse#UNKNOWN_GUILD_WELCOME_SCREEN Unknown Guild Welcome Screen}
```

Comment:
I wonder if we should convert this error into a `null` instead of forcing users to handle it as a user.